validate_rpm_specs.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
+
+import argparse
+import glob
+import os
+import sys
+
+from lib import exception
+from lib.utils import is_package_installed
+from lib.utils import run_command
+
+def validate_rpm_spec(spec_file_path):
+
+    try:
+        run_command("rpmlint -v %s" % spec_file_path)
+    except exception.SubprocessError as e:
+        #pylint: disable=no-member
+        print("validation of rpm spec file %s failed, output: %s" % (spec_file_path, e.stdout))
+        return
+
+def validate_rpm_specs(base_dir):
+    """
+    Validate specification files of rpm packages built by Open Power Host OS
+    """
+
+    packages = os.listdir(base_dir)
+
+    for package in packages:
+        files = glob.glob("{base_dir}/{package_name}/*/*/{package_name}.spec".format(
```

Comment:
Since `base_dir` is passed, I think the code should recursively search by all `*.spec` files under `base_dir` directory and validate them all.  Thus, if directory structure changes in the future, that won't break this function.